### ISSUE TITLE
Root type class com.intellij.openapi.roots.JavadocOrderRootType not found in PyCharm

### ISSUE THREAD

**Issue by kandeshvari** [KronicDeth/intellij-elixir#934]
# Version

6.5.1

# Exception

## Message

Root type class com.intellij.openapi.roots.JavadocOrderRootType not found. All roots: []

## Stacktrace

```
java.lang.AssertionError: Root type class com.intellij.openapi.roots.JavadocOrderRootType not found. All roots: []
	at com.intellij.openapi.roots.OrderRootType.getOrderRootType(OrderRootType.java:126)
	at com.intellij.openapi.roots.JavadocOrderRootType.getInstance(JavadocOrderRootType.java:35)
	at org.elixir_lang.mix.runner.MixRunningStateUtil.updateDocumentationPaths(MixRunningStateUtil.java:134)
	at org.elixir_lang.mix.runner.MixRunningStateUtil.updateRoots(MixRunningStateUtil.java:104)
	at org.elixir_lang.mix.runner.MixRunningStateUtil.updatedCodePaths(MixRunningStateUtil.java:66)
	at org.elixir_lang.mix.runner.MixRunningStateUtil.prependCodePaths(MixRunningStateUtil.java:51)
	at org.elixir_lang.mix.runner.MixRunningStateUtil.setElixir(MixRunningStateUtil.java:297)
	at org.elixir_lang.mix.runner.MixRunningStateUtil.commandLine(MixRunningStateUtil.java:181)
	at org.elixir_lang.annotator.Credo.generalCommandLine(Credo.java:107)
	at org.elixir_lang.annotator.Credo.generalCommandLine(Credo.java:149)
	at org.elixir_lang.annotator.Credo.generalCommandLine(Credo.java:268)
	at org.elixir_lang.annotator.Credo.doAnnotate(Credo.java:246)
	at org.elixir_lang.annotator.Credo.doAnnotate(Credo.java:43)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.d(ExternalToolPass.java:212)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.access$500(ExternalToolPass.java:47)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.run(ExternalToolPass.java:159)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:337)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:327)
	at com.intellij.util.ui.update.MergingUpdateQueue.lambda$flush$1(MergingUpdateQueue.java:277)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:291)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:246)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:246)
	at com.intellij.util.Alarm$Request.runSafely(Alarm.java:417)
	at com.intellij.util.Alarm$Request.access$700(Alarm.java:344)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:384)
	at com.intellij.util.Alarm$Request.run(Alarm.java:395)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:242)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:212)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

```

**Comment by KronicDeth** [comment]
That root system is super error prone in the Small IDEs.  I really wish JetBrains would make it not throw exceptions if a root type isn't supported.

**Comment by kandeshvari** [comment]
I have a silly question - why we need `JavadocOrderRootType` in elixir-plugin ?

**Comment by KronicDeth** [comment]
@kandeshvari because `JavadocOrderRootType` is the default documentation root type since there was no generic documentation root type for a long time that would work with the other parts of OpenAPI.

**Comment by KronicDeth** [comment]
Fixed by #955's c7192a8.  Released in 7.1.0.
